### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by tanaes**
This fixes a problem where unit tests weren't running because of no __init__.py, and associated drift. :) @mortonjt @qiyunzhu your code was slightly affected... please review/merge??

### CODE REVIEW

File: oecophylla/util/tests/test_parse.py [hunk 1]
```diff
@@ -138,73 +136,11 @@ def test_extract_sample_paths_db(self):
 
         self.assertDictEqual(d, exp_d)
 
-
-    def test_add_filter_db(self):
```
Comment (tanaes) [inline oecophylla/util/tests/test_parse.py:142]: We had gotten rid of this function a while ago...

File: oecophylla/util/parse.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def extract_sample_paths(seq_dir):
 
 
 # Option 2: read samples from sample sheet
-def read_sample_sheet(f, sep='\t', comment='#'):
```
Comment (tanaes) [inline oecophylla/util/parse.py:119]: Illumina sample sheets are actually csv, which is how I found this...

File: oecophylla/taxonomy/tests/test_parser.py [hunk 3]
```diff
@@ -13,7 +13,7 @@
                                         pandas2biom)
 
 
-class ParserTest(TestCase):
```
Comment (tanaes) [inline oecophylla/taxonomy/tests/test_parser.py:16]: changed for clarity with other TestParse class...

**Comment by tanaes** [comment]
@qiyunzhu @mortonjt want to merge this? Have another set of minor fixes coming...

File: oecophylla/util/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```
Comment (mortonjt) [inline oecophylla/util/__init__.py:1]: Do we really want this?  should the version number go in the main __init__.py?

**Review by mortonjt — APPROVED** [review]
Minor comment - otherwise good to go

**Comment by qiyunzhu** [comment]
This looks good to me. Let me just merge.

File: oecophylla/util/__init__.py (hunk 4 above)
Comment (tanaes) [reply oecophylla/util/__init__.py:1]: I have no idea how version number inits go. :) Just copied from other directory...

**Pull request merged**